docs/source/quickstart.rst

```diff
@@ -13,8 +13,13 @@ To create a Dask-CUDA cluster using all available GPUs and connect a Dask.distri
     from dask_cuda import LocalCUDACluster
     from dask.distributed import Client
 
-    cluster = LocalCUDACluster()
-    client = Client(cluster)
+
+    if __name__ == "__main__":
+        with LocalCUDACluster() as cluster:
+            with Client(cluster) as client:
+	        pass
+
+The `__main__` guard is required when running in Python script mode (instead of a notebook environment).
```

Comment:
Think my preference here would be to keep this code block as is, but make it explicit that this would be running from the context of a Python console (Distributed's [quickstart](https://distributed.dask.org/en/latest/quickstart.html#setup-dask-distributed-the-easy-way) for reference), and then include a note here linking to Distributed's upstream docs on [how to run a cluster from a Python script](https://docs.dask.org/en/stable/scheduling.html?highlight=__name__%20__main__#standalone-python-scripts).

Reply:
Thank you for the suggestion, I tried to mimic the dask doc. Along with which, the following warnings are fixed:
```
/home/jiamingy/Workspace/XGBoost-dev/dask-cuda/docs/source/examples/best-practices.rst:13: WARNING: Bullet list ends without a blank line; unexpected unindent.
/home/jiamingy/Workspace/XGBoost-dev/dask-cuda/docs/source/examples/ucx.rst:125: WARNING: Title underline too short.

``dask cuda worker`` with Manual Configuration
------------------------------------------
/home/jiamingy/Workspace/XGBoost-dev/dask-cuda/docs/source/examples/ucx.rst:125: WARNING: Title underline too short.

``dask cuda worker`` with Manual Configuration
------------------------------------------
/home/jiamingy/Workspace/XGBoost-dev/dask-cuda/docs/source/ucx.rst:2: WARNING: Duplicate explicit target name: "here".
```